TestAssets/TestProjects/docker-compose/docker-compose.dcproj

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" Sdk="Microsoft.Docker.Sdk">
+  <PropertyGroup Label="Globals">
+    <ProjectVersion>1.1</ProjectVersion>
+    <ProjectGuid>380948fd-dd78-4d21-94f2-055bd88886e5</ProjectGuid>
```

Comment:
This is added by CPS in VS. We don't really need it. But again, this is a DockerCompose project instead of a .NET Core project. I could remove it from the test though.